src/scripts/libs/create-marker.ts

```diff
@@ -31,7 +31,7 @@ export async function createMarker(marker: Marker): Promise<Entity> {
 
       resolve(
         new Entity({
-          id: `${marker.id}`,
+          id: marker.id && `${marker.id}`,
```

Comment:
does this work without any id? if not we could set the title to have something uniq as the id

Reply:
Yes it creates a unique id automatically if none is set.